### ISSUE TITLE
[디자인] 웹 접근성을 위한 명암비

### ISSUE THREAD

**Issue by eunbin20** [eunbin20/til#17]
## Luminance

휘도, 또는 밝기. 요소의 밝기 정도를 나타낸다.

Luminance 계산법 - https://www.w3.org/TR/WCAG20/#relativeluminancedef

```jsx
const luminance = 0.2126 * R + 0.7152 * G + 0.0722 * B
```

## Contrast Ratio

명암비. 두 색상의 밝기의 차이에 대한 비율값. 

Contrast ratio 계산법 - https://github.com/tmcw/wcag-contrast/blob/master/index.js

```jsx
 const getColorContrast = (luminance1, luminance2) => {
   const brighter = Math.max(luminance1, luminance2);
   const darker = Math.min(luminance1, luminance2);

   return (brighter + 0.05) / (darker + 0.05);
 }
```

- Contrast ratio 권장 기준값
  -  7.0 이상: AAA
  - 4.5 이상이면서 폰트 사이즈가 14pt+bold 혹은 18pt 보다 큼: AAA Large
  - 4.5 이상: AA 
  - 3.0 이상: AA Large
  - 2.9 미만: Fail
  
WCGA에서는 명암비가 4.5:1 이상의 대비를 가진 요소를 권장하고있다.
